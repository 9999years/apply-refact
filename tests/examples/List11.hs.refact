[("tests/examples/List11.hs:1:17: Warning: Use String\nFound:\n  Maybe [Char]\nWhy not:\n  Maybe String\n",[Replace {rtype = Type, pos = SrcSpan {startLine = 1, startCol = 23, endLine = 1, endCol = 29}, subts = [], orig = "String"}])]
