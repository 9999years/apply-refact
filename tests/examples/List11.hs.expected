data Yes = Yes (Maybe [Char])
